Index.html

```diff
@@ -0,0 +1,313 @@
+                <li class="multcss">css</li>
+                <li class="multJava">JavaScript</li>
+              </ul>
+              <div class="buttonmult">
+                <button type="submit" class="go-buttonmult">See project</button>
+              </div>
+            </li>
+          </li>
+        </ul>
+      </nav>
+    </main>
+    <section class="contact-me">
+      <div class="head-section-co">
+        <div id="head-container-co">
+          <h2 class="head-co">
+            About <br />
+            Myself
+          </h2>
+          <p class="description-co">
+            Hello I’m a software developer! I can help you build a product,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <h2 class="connect-co">LET’S CONNECT</h2>
+          <ul class="social-media-icons">
+            <li>
+              <a href="#" target="_blank">
+                <img src="images/github.png" alt="git" />
+              </a>
+            </li>
+            <li>
+              <a href="#" target="_blank">
+                <img src="images/Linkedin.png" alt="linkedIn" />
+              </a>
+            </li>
+            <li>
+              <a href="#" target="_blank">
+                <img src="images/angelite.png" alt="image" />
+              </a>
+            </li>
+            <li>
+              <a href="#" target="_blank">
+                <img src="images/twitter.png" alt="image" />
+              </a>
+            </li>
+            <li>
+              <a href="#" target="_blank">
+                <img src="images/medium.png" alt="image" />
+              </a>
+            </li>
+          </ul>
+          <div class="new-code">
+            <button type="submit" class="go-newmes">My resume</button>
+          </div>
+          <ul class="skill">
+            <li class="event12">
+              <li class="event7">
+                <h3 class="languages">Languages</h3>
+                <button type="submit" class="dropdown">
+                  <img src="./images/Icon-Chevron-Down-Larger.png" alt="" />
+                </button>
+              </li>
+              <li>
+                <ul class="language-list">
+                  <li class="JavaScript1">
+                    <div class="lang">
+                      <img src="./images/Ellipse-1.png" alt="" />
+                      <p>JavaSript</p>
+                    </div>
+                  </li>
+                  <li class="HTML1">
+                    <div class="lang">
+                      <img src="./images/Ellipse-2.png" alt="" />
+                      <p>HTML</p>
+                    </div>
+                  </li>
+                  <li class="CSS1">
+                    <div class="lang">
+                      <img src="./images/Ellipse-3.png" alt="" />
+                      <p>CSS</p>
```

Comment:
- [ ] Please 🙏🏽 use the correct font styles to be an exact duplicate of the Figma template 👇🏽 

![image](https://github.com/jovasal/Finish-Mob-Figma/assets/58642949/8f70746d-95ea-4f70-821b-bdc693b5560e)

Reply:
I tried to use only a font family Poppois, but is not working. Each time that I tried to use it alone, Linter errors show:
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/6005c467-a8ba-4f0b-b6e6-a6e9771d3bfc)
and this error comes to:
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/d6ba6ce4-c20c-44ed-93b4-b3fe1e5b0035)
for this reason, the code is using (font-family:  "Poppins", sans-serif;) that is too similar:
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/f56949db-154e-4358-9408-9693855ea1c4)

On the other hand, Code contains the exact information in Figma template as you can see in the following image:
![image](https://github.com/jovasal/Finish-Mob-Figma/assets/35693330/8aafad14-ca21-4ba9-aa65-2f86ca1b0d95)